task/backend/scheduler_metrics.go

```diff
@@ -71,12 +80,14 @@ func (sm *schedulerMetrics) PrometheusCollectors() []prometheus.Collector {
 		sm.runsActive,
 		sm.claimsComplete,
 		sm.claimsActive,
+		sm.queueDelta,
 	}
 }
 
 // StartRun adjusts the metrics to indicate a run is in progress for the given task ID.
-func (sm *schedulerMetrics) StartRun(tid string) {
+func (sm *schedulerMetrics) StartRun(tid string, queueDelta float64) {
```

Comment:
Add godoc and maybe rename the new parameter to reflect that it is in seconds.

We do have tests for the scheduler metrics. I don't very well understand how prometheus histograms work, but it wouldn't hurt to add a test for it.

Reply:
Ill update the godoc, and also look into the test. good eye :)